PokemonTabletopAdventures_v3/PTA3.html

```diff
@@ -2757,4 +2783,223 @@ <h2>Configuration Options</h2>
+      if(innerData["Nature"]) response["nature-type"] = innerData["Nature"].split("[")[0].toLowerCase();
+      remapIfExists(innerData, response, "Size", "size");
+      remapIfExists(innerData, response, "Weight", "weight");
+      remapIfExists(innerData, response, "Egg1", "egg_group");
+      if(innerData["Egg2"]) response["egg_group"] += "/" + innerData["Egg2"];
+      remapIfExists(innerData, response, "Diet", "diet");
+      remapIfExists(innerData, response, "Type_1", "type1");
+      if(innerData["Type_1"]) response["type1"] = innerData["Type_1"].toLowerCase();
+      if(innerData["Type_2"]) response["type2"] = innerData["Type_2"].toLowerCase();
+      remapIfExists(innerData, response, "Held_Item", "item");
+
+      characterData["Moves"]?.forEach(function(move) {
+        const id = generateRowID();
+        remapIfExists(move, response, "Name", `repeating_moves_${id}_move_name`);
+        remapMoveFrequency(move, response, id);
+        remapIfExists(move, response, "Range", `repeating_moves_${id}_move_range`);
+        if(move["num_dice"]) response[`repeating_moves_${id}_move_dicenumber`] = parseInt(move["num_dice"]) || 0;
+        if(move["die_size"]) response[`repeating_moves_${id}_damage_dice`] = move["die_size"].substring(1);
+        if(move["Type"]) response[`repeating_moves_${id}_move_type`] = (move["Type"] == "No-Type" ? "typeless" : move["Type"].toLowerCase());
+        remapMoveCategory(move, response, id);
+        remapIfExists(move, response, "Effect", `repeating_moves_${id}_move_effect`);
+      });
+
+      if(characterData["Stats"]) {
+        const innerData = characterData["Stats"];
+        remapIfExists(innerData, response, "Max_HP", "HP_max");
+        remapIfExists(innerData, response, "Max_HP", "HP");
+        const natureModifiers = createNatureAttrs(response["nature-type"]); // Returns 0 array if doesn't exist or not a typical nature
+        if(innerData["ATK"]) response["atkbase"] = innerData["ATK"] - natureModifiers["atknature"];
+        if(innerData["DEF"]) response["defbase"] = innerData["DEF"] - natureModifiers["defnature"];
+        if(innerData["SPATK"]) response["spatkbase"] = innerData["SPATK"] - natureModifiers["spatknature"];
+        if(innerData["SPDEF"]) response["spdefbase"] = innerData["SPDEF"] - natureModifiers["spdefnature"];
+        if(innerData["SPD"]) response["spdbase"] = innerData["SPD"] - natureModifiers["spdnature"];
+      }
+
+      remapIfExists(characterData, response, "Stat_Passives", "statpassives");
+      remapIfExists(characterData, response, "Non-Stat_Passives", "otherpassives");
+      remapIfExists(characterData, response, "Skills", "pokemonskills");
+
+      response["json_import_response"] = "Pokemon data successfully imported from " + source + "!";
+      return response;
+
+    } else if (source.startsWith("roll20")) {
+      // Identify as a roll20 sheet
+      let response = includeOnlyKeysInArray(characterData, r20export_attrs);
```

Comment:
Another one that can probably be a const.
```suggestion
      const response = includeOnlyKeysInArray(characterData, r20export_attrs);
```